layers/pct.py

```diff
@@ -0,0 +1,54 @@
+from utils import *
+
+
+def makePct():
+    tmp = "tmp_pct"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    run(
+        [
+            "wget",
+            "https://github.com/npct/pct-outputs-national/raw/master/commute/lsoa/rnet_all.geojson",
+            "-O",
+            f"{tmp}/commute.geojson",
+        ]
+    )
+
+    run(
+        [
+            "wget",
+            "https://github.com/npct/pct-outputs-national/raw/master/school/lsoa/rnet_all.geojson",
+            "-O",
+            f"{tmp}/school.geojson",
+        ]
+    )
+
+    # The two trip purposes are split into different files, and neither feature
+    # ID nor the local_id property matches between them. So just output two
+    # separate files.
+
+    def fixProps(inputProps):
+        return {
+            "baseline": int(inputProps["bicycle"]),
```

Comment:
@robinlovelace-ate, am I picking up the correct fields? That output repo doesn't have / link to definitions for things, so I'll compare to carbon.place later manually

Reply:
Yep!

Reply:
There are definitions burried in there...

Reply:
Here are the codebooks: https://github.com/npct/pct-shiny/tree/master/regions_www/www/static/02_codebooks/commute